websites/api-docs-smoke-test/source-raml/test/types/unions.raml

```diff
@@ -3,25 +3,28 @@ usage: Unions test types.
 
 # Not supporting actual RAML Union types, only a pattern that has a parent without own properties except the discriminator
 
+uses:
+  enums: enums.raml
+
 types:
   UnionLikeType:
     description: |
       A pattern of inheritance that is to be represented like a union, i.e. this parent type would not be shown with a property list on its own.
     type: object
     discriminator: kind
     properties:
-      kind: string
+      kind: enums.EnumWithDescriptionExample
```

Comment:
Can you set up the examples of the constantLike in a way that is independent of the UnionLike and cover a case where the type of the constant is not a string but for example an integer?